jwst/associations/lib/rules_level3_base.py

```diff
@@ -294,6 +299,56 @@ def make_member(self, item):
         }
         return member
 
+    def make_fixedslit_bkg(self):
```

Comment:
Hmmm, apparently I forgot to save my previous commentary. Let's try this again....

If I have this right, the logic of this routine is basically is that, for each science member, add it as a background member and rename to 'x1d'.  The idea is that all the science should also be considered backgrounds in the processing.

If that is right, then a suggested refactor: The current routine is rebuilding the association from scratch. All that is necessary is to append the new backgrounds to the current association.

And hitting the save button...

Reply:
refactored

Reply:
Nice! A few more comments below...